### ISSUE TITLE
Filebrowser and Photoprism not accessible with Docker installed

### ISSUE THREAD

**Issue by Kurisutian** [openmediavault/openmediavault#1417]
**Describe the bug**

When installing either of the plugins, Filebrowser and/or Photoprism and installing Docker will lead to a situation where the web interface of both plugins cannot be accessed. This has been also discussed in the Forum where a workaround was described.
When this happens both plugins as well as Docker need to be uninstalled, the system has to be rebooted and after that the plugins need to be installed first followed by Docker afterwards. If at any time the system now will be rebooted the Plugins again are not accesible and the procedure needs to be repeated. 

**To Reproduce**

1. Install Filebrowser and/or Photoprism as Plugin and activate them.
2. Install Docker
3. Reboot OMV
4. Test accessing the plugins web frontend. 

**Expected behavior**

The webfrontends of the installed and activated plugins is opening in the browser.

**Reference to Forum**

https://forum.openmediavault.org/index.php?thread%2F42564-filebrowser-not-accessible%2F=&pageNo=1

**openmediavault Server (please complete the following information):**

- OS version: Linux homenas 5.19.0-0.deb11.2-amd64 #1 SMP PREEMPT_DYNAMIC Debian 5.19.11-1~bpo11+1 (2022-10-03) x86_64 GNU/Linux
- openmediavault version: 6.0.46-4

**Client (please complete the following information):**

- Device: Linux Desktop PC
- OS: Manjaro Linux
- Browser: Firefox, Chrome
- Version: Both the latest available

**Additional context**

**Comment by ryecoaaron** [comment]
I can't replicate this on any of my systems.  I would guess there is something odd with a few peoples setups but I can't tell from the thread.  I highly doubt it is podman or OMV causing the problem.  So, this doesn't belong in the OMV issues.  If anything, I would say docker is causing it and *maybe* omv-extras should change something.  But if I can't recreate it, I can't "fix" it.

**Comment by votdev** [comment]
I am using Docker, podman and k3s in parallel on my system without any issues.

**Comment by Kurisutian** [comment]
I just installed OVM extras with docker and yacht. But all according to the documentation and nothing was changed our configured through anything but the UI in OMV.
If I can provide any specific files or can help in any way to investigate this I will gladly do so. I just need to know what is needed from my end.

**Comment by Kurisutian** [comment]
​So after some more searching and finding a solution for Photoprism not being accessible at all (also if I do not have Docker installed to OMV) I came across this thread: https://forum.openmediavault.org/index.php?thread%2F43102-photoprism-not-accessible-from-local-network%2F=
​
​It seems that Docker and Podman are using the same internal network and this causes the problem with the Plugins. I still cannot access Photoprism (which seems to be another issue) but after changing the subnet of Podman as suggested in this thread, I was able to still access Filebrowser even after docker was installed and the system was restarted. ​

@votdev do you have different subnets for Podman and Docker? That would explain the different behaviour.

**Comment by votdev** [comment]
> @votdev do you have different subnets for Podman and Docker? That would explain the different behaviour.

I can not check that anymore because i've fully migrated to podman and removed Docker.

**Comment by Kurisutian** [comment]
> > @votdev do you have different subnets for Podman and Docker? That would explain the different behaviour.
> 
> I can not check that anymore because i've fully migrated to podman and removed Docker some time ago. But i can say that i did not change anything related to networking configuration of podman and Docker.

I have tested it here by simply making the change and rebooting several times. Filebrowser was still reachable and running fine, as well as all docker containers. As far as I can tell, this did "fix" the issue for good.
